packages/firestore/console.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Local console build for
+declare var firebase;
+
+import './src/platform_browser/browser_init';
+import { Firestore } from './src/api/database';
+import { configureForFirebase } from './src/platform/config';
+
+export function registerFirestore(instance) {
+  configureForFirebase(instance);
+}
+
+registerFirestore(firebase);
```

Comment:
This seems to be identical to index.ts except it expects `firebase` to be an existing global, but I don't think this is actually what console needs.

`firebase` will _not_ exist in the console (or if it is, it isn't what we want) and we need to expose our public constructor to the console environment somewhere (see goog_module_config.ts for how we did this previously).

So unless I'm confused, this isn't going to work for console folks.  If you haven't already, I'd work with the console folks to get an end-to-end solution working before sending this out before trying to get this checked in...  and it would be great if you could document how the integration works somewhere (how to trigger this build, what it exposes, how console consumes it, etc.).

Reply:
Was up working with @samhorlbeck and was under the impression that FB existed and that this was all that was needed. I'll sync back up w/ him and @tstirrat.

Reply:
I could be confused.  As long as we have a (manual) end-to-end "build from github => integrate into console => console still works" validation, then I'll be happy. :-)